src/include/cell.i

```diff
@@ -988,8 +987,17 @@ __cell_unpack_window_cleanup(WT_SESSION_IMPL *session, const WT_PAGE_HEADER *dsk
         if (ta->newest_stop_txn != WT_TXN_MAX) {
             ta->newest_stop_txn = WT_TXN_NONE;
             F_SET(unpack_addr, WT_CELL_UNPACK_TIME_WINDOW_CLEARED);
-            if (ta->newest_stop_ts == WT_TS_MAX)
+
+            /*
```

Comment:
I didn't fully understand this part.

> WT_TS_MAX is not considered for aggregated durable timestamp calculation

Do we mainly care about this for garbage collection reasons?

Also minor nitpick but I've been told not to put variables or constant names in comments. So if this could be reworded to avoid that, it'd be better.

Reply:
While calculating the aggregated durable timestamps, WT_TS_MAX is considered in that calculation.  I updated the comments that may be easy to understand. The reason behind not considering the WT_TS_MAX as part of durable timestamps is to find out whether the btree has more data than the stable timestamps for rollback to stable.